src/core/RenderVariables.cc

```diff
@@ -0,0 +1,22 @@
+#include "RenderVariables.h"
+#include "Context.h"
+#include "BuiltinContext.h"
+#include "Value.h"
+
+void
+RenderVariables::setRenderVariables(ContextHandle<BuiltinContext>& context) const
```

Comment:
I'd name this `applyToContext()` or smth. to make it more clear what it does.

Reply:
Yes